context.go

```diff
@@ -779,35 +790,25 @@ func (c *Context) RemoteIP() (net.IP, bool) {
 		return nil, false
 	}
 
-	if c.engine.trustedCIDRs != nil {
```

Comment:
You should keep `if c.engine.trustedCIDRs != nil {}` to provide a choice to disable `TrustedProxies` feature.

Reply:
@Bisstocuz such check is already in `isTrustedProxy()`: context.go:769 . Are you suggesting to copy this check to have in both places?

Reply:
@agmt Sorry, I got missed for that.